src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -243,6 +247,21 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 3,
                 "No actions matched the current request. Route values: {RouteValues}");
+
+            _featureNotFound = LoggerMessage.Define<string, string>(
+                LogLevel.Debug,
```

Comment:
Warning because it's not a bug or a failure, but something you tried to do isn't going to work.